{"name": "test_01_login_error[data0]", "status": "passed", "attachments": [{"name": "log", "source": "04a332e6-6ebe-4264-903d-ea18a4e23d32-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'Caseid': 1, 'module': '登录', 'title': '不输入用户名', 'username': '', 'password': '123654', 'expect': '请输入手机号'}"}], "start": 1592286142678, "stop": 1592286153304, "uuid": "8778e7da-2209-4aac-aadc-54322235ae5e", "historyId": "a1e027ef5dd8cfe3735eda6d86eb167f", "fullName": "Cases.test_login#test_01_login_error[data0]", "labels": [{"name": "tag", "value": "error"}, {"name": "parentSuite", "value": "Cases"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "Danny"}, {"name": "thread", "value": "14124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Cases.test_login"}]}